arch/x86/core/x86_mmu.c

```diff
@@ -1317,7 +1317,7 @@ int arch_buffer_validate(const void *addr, size_t size, bool write)
 	int ret = 0;
 
 	/* addr/size arbitrary, fix this up into an aligned region */
-	k_mem_region_align((uintptr_t *)&virt, &aligned_size,
+	(void)k_mem_region_align((uintptr_t *)&virt, &aligned_size,
 			   (uintptr_t)addr, size, CONFIG_MMU_PAGE_SIZE);
```

Comment:
Don't you loose the correct alignment of the second line when the first line gets shifted forward by the `(void)`?

Reply:
I'm talking about the visual code alignment, that the split line begins right after the opening parenthesis of the first line.

Reply:
> In general I limit my changes to the minimum number of lines and I avoid to change indentation of continuation line as this is not uniform in code.

Yeah, I know that there are inconsistencies across the project, but at least this should be kept consistent in a single c-file or submodule. You're effectively introducing new breakage to one kind of coding issues (split line alignment) while fixing another kind of coding issues (the MISRA stuff). Anyway, I already approved, i.e. this is not a deal-breaker to me - I was just surprised that you're ok with introducing this new kind of issues while fixing other ones.

Reply:
Perhaps I have not explained well what I meant: as I don't know what is the indentation rule for continuation lines and I've not been able to deduce it from code as it is not uniform, I've preferred to not change the line.